src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisController.java

```diff
@@ -82,10 +84,9 @@
 			.of(BuiltInAnalysisTypes.PHYLOGENOMICS, "tree", BuiltInAnalysisTypes.SISTR_TYPING, "sistr",
```

Comment:
`PREVIEWS` is no longer used, are you going to get ride of it?

Reply:
Yup it's on my list of stuff to remove once I've replaced it